### ISSUE TITLE
No option to reset despite implication there is ? 

### ISSUE THREAD

**Issue by erdelf** [pardeike/RimWorld4Debugging#7]
**Describe the bug**
On Step 11, on how to revert the changes, it is said that one "can then simply go back to step 9 to patch it again".
Yet going back to that step, the button to do step 9 is not present anymore, and only the "Next" button is visible.

A restart of the program doesn't fix it, so the only solution currently seems to be to delete the cache files.

**To Reproduce**
1. Complete the process.
2. Reset RimWorld
3. Go back to step 9

**Expected behavior**
Following the description, I would expect an easily recognizable as such method to reset the progress (?), and repeat the step.

**Screenshots**
![image](https://github.com/pardeike/RimWorld4Debugging/assets/19973727/ec47cd6b-fc72-42dc-95f9-ac66243c4290)
![image](https://github.com/pardeike/RimWorld4Debugging/assets/19973727/45cd06d7-bfad-4832-8a11-71f95b8cc456)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Electron

**Comment by pardeike** [comment]
Yeah, I have seen this problem. I think I even toyed around removing the disabled state and the prefs for that but have not cleaned it up for a release. Will look into that.
